web-console/knikubevirt/expose-vm-as-a-service/expose-vm-as-a-service.md

```diff
@@ -0,0 +1,49 @@
+# Expose VM as a Network Service
+
+The user should be able to expose a VM as a network service. This means that they would be exposing some of the ports on its default interface to the cluster internally or to the outside world. 
+In order for it to be actually useful, we need to bind it to a service.
+
+## Expose From VM List
+
+![Expose From VM List](img/Expose_01.png)
+
+We added a new action called “Expose VM as network service”. Clicking on that will open the expose as network popup. 
+
+![Pop up](img/Expose_02.png)
```

Comment:
Just checking if I'm reading this form correctly. Is the IP always required or is this form using progressive disclosure to indicate some indented options are only required when certain options are selected from the dropdown?

Reply:
IP is only if Cluster IP is selected so it would be a progressive disclosure case.

Reply:
@yfrimanm - It might make more sense to just generally note that depending on which Type is chosen, then options will change. Cluster IP is just one example. Also, it probably doesn't belong here. I think the note belongs where you say "The user can select between 3 Network Service types..."